src/main/resources/db/migration/V80__vedtaksresultat_Innvilget_med_opphoer_utgaar.sql

```diff
@@ -0,0 +1 @@
+update vedtak set resultat_type = 'INNVILGE' where resultat_type = 'INNVILGE_MED_OPPHØR';
```

Comment:
Fint å huske på denne 👏

Reply:
Er egentlig bare relevant for preprod (siden det er featuretogglet i prod), men kjekt å rydde opp etter seg der også :)